flexget/components/jellyfin/from_jellyfin.py

```diff
@@ -0,0 +1,89 @@
+from loguru import logger
+
+from flexget.components.jellyfin.api_jellyfin import JellyfinAuth, JellyfinApi
+from flexget.config_schema import one_or_more
+from flexget import plugin
+from flexget.event import event
+from flexget.components.jellyfin.jellyfin_util import SCHEMA_SERVER_TAG, SORT_FIELDS
+
+logger = logger.bind(name='from_jellyfin')
+
+
+class JellyfinInput:
+    """
+    Returns Jellyfin Inputs
+
+    Example:
+        from_jellyfin:
+            server:
+                host: http://localhost:8096
```

Comment:
Looks like we have a bit of a mess how we configure these in external application plugins. Some plugins use `uri`, some plugins use separate `host` and `port` keys. Couple use even more esoteric `base_url`.

Using `host` here and passing uri/url into it would be another mix. Let's reserve it when used in format: 

```
host: localhost
port: 1234
```

Please change the key to `uri` or split into separate `host` and `port` keys.

Reply:
Make sense, I will wait for emby plug-in to be updated and make the changes after that

> Example:
 from_emby:
            server:
                host: http://localhost:8096